syntaxes/gdb.tmLanguage.json

```diff
@@ -214,6 +214,13 @@
 						"2": { "name": "constant.numeric.count.ignore.gdb" }
 					}
 				},
+				{
+					"name": "meta.command.dont.gdb",
```

Comment:
I prefer to use the full command name for "name" field, to avoid ambiguities so `meta.command.dont-repeat.gdb`